### DISCUSSION TITLE
We now have support for Hatch! 🎉

### DISCUSSION THREAD

**Discussion by DavidVujic** [DavidVujic/python-polylith#158] [Announcements]
The Python tools for Polylith now has support for Hatch 🎉 

In addition to the existing Poetry plugin, that already enable tooling support for Polylith, there is now also a CLI tool with the same kind of interface as the plugin. The new CLI aims to support several Packaging & Deployment Management tools, and initially supporting `hatch`. You can also use it for Poetry projects, even if the recommended way is to use the Poetry plugin as before.

To make the tooling fully support Hatch, there is also a Build Hook Plugin available - `hatch-polylith-bricks` - that is useful when building & packaging libraries from a Polylith workspace. It includes the same kind of feature as `poetry build-project --with-top-namespace` does.

Read more about it in the docs: https://davidvujic.github.io/python-polylith-docs/

There is also a 6-minute intro video about the new Hatch support available on YouTube: https://youtu.be/K__3Uah3by0

**Comment by GH-maggio** [comment]
@DavidVujic wow this is great, thanks for this feature!
